src/components/Human.tsx

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable no-nested-ternary */
+import classNames from 'classnames';
+import { useParams } from 'react-router-dom';
+import { Person } from '../types';
+import { PersonLink } from './PersonLink';
+
+type Props = {
+  person: Person;
+};
+
+export const Human: React.FC<Props> = ({
+  person,
+}) => {
+  const {
+    sex,
+    born,
+    died,
+    motherName,
+    fatherName,
+    mother,
+    father,
+  } = person;
+
+  const { slug } = useParams();
+  const selectedPersonSlug = slug || '';
```

Comment:
```suggestion
  const { slug = '' } = useParams();
```
You can use the default value instead of the `or` operator